test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -145,6 +145,7 @@
     ("aten::_weight_norm_cuda_interface_backward", datetime.date(9999, 1, 1)),
     ("aten::segment_reduce", datetime.date(9999, 1, 1)),
     ("aten::_segment_reduce_backward", datetime.date(9999, 1, 1)),
+    ("aten::empty.SymInt", datetime.date(9999, 1, 1)),
```

Comment:
```suggestion
    ("aten::empty.SymInt", datetime.date(2022, 7, 1)),
```
Should we be setting more reasonable dates here?

Reply:
when do we set the dates to that versus a normal date?

Reply:
It sets a time-bomb for reverting the revert by Jul 1st, which I don't think anyone promises will happen